index.html

```diff
@@ -1,15 +1,341 @@
+                    <a href="#">
+
+                        <figure><img class="populaire-room"
+                                src="./assets/images/hebergements/4_small/febrian-zakaria-sjvU0THccQA-unsplash.jpg"
+                                alt="">
+                        </figure>
+                        <div class="populaire-info">
+                            <h4>Hôtel Tout bleu et Blanc</h4>
+                            <p>Nuit à partir de 68€</p>
+
+                            <ul>
+                                <li><i class="fas fa-star"></i></li>
+                                <li><i class="fas fa-star"></i></li>
+                                <li><i class="fas fa-star"></i></li>
+                                <li><i class="fas fa-star"></i></li>
+                                <li><i class="fas fa-star"></i></li>
+
+                            </ul>
+                        </div>
+
+                    </a>
+                </article>
+        </section>
+<!--activities section-->
+        <section id="activity">
+            <article>
+                <a href="#">
+                    <figure>
+                        <img src="./assets/images/activites/1_xlarge/reno-laithienne-QUgJhdY5Fyk-unsplash.jpg"
+                            alt="reno-laithienne">
+                        <figcaption>Vieux Port</figcaption>
+                    </figure>
+                </a>
+            </article>
+            <article>
+                <a href="#">
+                    <figure>
+                        <img src="./assets/images/activites/2_large/paul-hermann-QFTrLdQIRhI-unsplash.jpg"
+                            alt="paul-hermann">
+                        <figcaption>Fort de Pomègues</figcaption>
+                    </figure>
+                </a>
+            </article>
+            <article>
+                <a href="#">
+                    <figure>
+                        <img src="./assets/images/activites/2_large/kevin-hikari-rV_Qd1l-VXg-unsplash.jpg"
+                            alt="kevin-hikari">
+                        <figcaption>Îles du Frioul</figcaption>
+                    </figure>
+                </a>
+            </article>
+            <article>
+                <a href="#">
+                    <figure>
+                        <img src="./assets/images/activites/1_xlarge/kilyan-sockalingum-NR8-cBCN3aI-unsplash.jpg"
+                            alt="kilyan-sockalingum">
+                        <figcaption>Parc National des Calanques</figcaption>
+                    </figure>
+                </a>
+            </article>
+            <article>
+                <a href="#">
+                    <figure>
+                        <img src="./assets/images/activites/2_large/florian-wehde-xW9e8gdotxI-unsplash.jpg"
+                            alt="florian-wehde">
+                        <figcaption>Notre-Dame-de-la-Garde</figcaption>
+                    </figure>
+                </a>
```

Comment:
close what ?

Reply:
i checked with HTML tester and corrected it